test/parallel/test-http-outgoing-end-cork.js

```diff
@@ -3,11 +3,11 @@ const common = require('../common');
 const assert = require('assert');
 const http = require('http');
 
-const REQ_TIMEOUT = 500; // Set max ms of request time before abort
+const REQ_TIMEOUT = common.platformTimeout(500); // Set max ms of request time before abort
```

Comment:
I think it does nothing (it just returns 500) for the riscv64 platform.

Reply:
That's true 😞 I opened this PR up mainly because I read this:

> but it would be nice anyway to have timeouts proportional to platformTimeout for all tests.

For the actual test timeout - would increase the timeout for `riscv64` help? If so, how much we should increase?

Reply:
Usually riscv64 is very slow, so I'd set a bigger multiplier.

Reply:
I tried to add support for `riscv64` in `platformTimeout` and increase the default to `multipliers.four`, not sure if it is on the right track, PTAL @lpinca @kapouer 🙏

Reply:
LGTM